data/services/cdn-route.toml

```diff
@@ -1,3 +1,3 @@
 name = "cdn-route"
-description = "Custom routes with HTTPS certificates using Amazon CloudFront"
+description = "Custom domains, CDN caching, and TLS certificates using Amazon CloudFront"
```

Comment:
The `.toml` files are what is used in the HTML table/pages and can be HTML/markdown content. The `description` in the markdown files are actually page metadata and should **not** contain HTML/markdown content.

The content _could_ be shared in jekyll, so I suspect there's a way to do it in hugo (although I'm not familiar with it). However, given that these fields contain different content types, I think it's better to keep them separate even if it means duplication.